package.json

```diff
@@ -1414,6 +1414,11 @@
             "type": "object",
             "required": [],
             "properties": {
+              "processCommand": {
+                "type": "string",
+                "description": "Attaches to a process with this command",
+                "default": "If this is use 'processId' and 'processName' should not be used."
```

Comment:
There is a typo here. You want 'used' instead of 'use'.

But thinking about it more, you might want to instead use something like `dotnet exec /example/path/to/program.dll`  to provide an example of the kind of thing we might want.